tests/session/cmd/test_parallel.py

```diff
@@ -186,6 +187,19 @@ def test_parallel_no_spinner(tox_project: ToxProjectCreator) -> None:
     )
 
 
+def test_parallel_no_spinner_ci(tox_project: ToxProjectCreator) -> None:
+    """Ensure spinner is disabled by default in CI."""
+    with mock.patch.object(parallel, "execute") as mocked, mock.patch.dict(os.environ, {"CI": "1"}):
```

Comment:
Use pytest-mock instead

Reply:
Fixed

Reply:
OK, I had to rolled back. It seems pytest-mock has some issue with `os.environ`. Also `unittest.mock` is used to modify `os.environ` throughout tox tests.

Reply:
I strongly urge you to reconsider. For patching the operating system environment variables, you should use the built-in pytest monkeypatch fixture.

Reply:
I will not accept any pull request that doesn't use the already practiced tools and introduces a new way of doing the same thing.